ui/src/app/store/machine/utils/storage.ts

```diff
@@ -284,7 +314,7 @@ export const isMounted = (fs: Filesystem | null): fs is Filesystem => {
   // VMware ESXi does not directly mount the partitions used. As MAAS can't
   // model that, a placeholder "RESERVED" is used for datastores so we know that
   // these partitions are in use.
-  return fs.mount_point !== "" && fs.mount_point !== "RESERVED";
+  return ![null, "", "RESERVED"].includes(fs.mount_point);
```

Comment:
Could `mount_point` also be undefined? Could this be `!!fs.mount_point && fs.mount_point !== "RESERVED";`?

Reply:
Yeah I think a catch-all for falsy values is probably better!